quipucords/scanner/input_log.py

```diff
@@ -0,0 +1,301 @@
+# Copyright (c) 2018 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 3 (GPLv3). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv3
+# along with this software; if not, see
+# https://www.gnu.org/licenses/gpl-3.0.txt.
+
+"""Log all input to the filesystem for later debugging."""
+
+import json
+import logging
+import multiprocessing as mp
+import os
+import time
+import uuid
+from django.db import transaction
+from api import log_state
+from quipucords import settings
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+UUID_CACHE = None
+
+DEFAULT_MAX_SIZE = 1 << 30  # 1 GiB
```

Comment:
How do you specify a different max size?
How large is the file after a single scan of our 21 systems?

Reply:
Moved this and max_age to `settings.py` and added the ability to override with an environment variable.

Reply:
I got about 2.3MB of log from scanning all of `10.10.[181:182].[0:255]`. That scan found 24 reachable IPs, so very roughly, 100KB/machine/scan.